python/llm/portable-installer/README.md

```diff
@@ -0,0 +1,12 @@
+# BigDL-LLM One-Click Installer For Windows
+
+## Supported models
+
+- ChatGLM2-6b
+- Baichuan-13B-Chat
+- Baichuan2-7B-Chat
```

Comment:
We could change the title to `Verified Models`, as I believe this one-click tool can work for much more models.

Reply:
Done.